ports/atmel-samd/common-hal/microcontroller/Pin.c

```diff
@@ -53,6 +57,11 @@ void reset_all_pins(void) {
     pin_mask[0] &= ~(PORT_PA31);
     #endif
 
+    for (uint32_t i = 0; i < PORT_COUNT; i++) {
+        pin_mask[i] &= ~(PORT_PA31);
```

Comment:
Why PORT_PA31, when this could be PB or PC, since you're iterating through all the ports?

Reply:
I think its a copy and paste error.